postgresql-dialect/src/commonMain/kotlin/app/cash/sqldelight/db/PgSqlDriver.kt

```diff
@@ -0,0 +1,36 @@
+package app.cash.sqldelight.db
+
+import app.cash.sqldelight.Query
+import app.cash.sqldelight.Transacter
+
+expect class PgSqlDriver : SqlDriver<PgSqlPreparedStatement, PgSqlCursor> {
```

Comment:
i suspect but am maybe wrong that `PgSqlDriver` does not need to be `expect`, only `PgSqlCursor` and `PhSqlPreparedStatement`, and if those ones only use other KMM libraries as their implementation then you dont need expect/actual anywhere

Reply:
What about for the implementation of `Closable`? On native, I believe we'd need to implement it with `PQfinish`. 

https://www.postgresql.org/docs/9.1/libpq-connect.html